tests/constraints-base.in

```
@@ -1,4 +1,4 @@
 # Known limitations for indirect/transitive dependencies.
 
-rstcheck < 6  # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
+rstcheck>=6.2.4
```

Keep the previously sparse style.
```suggestion
rstcheck >= 6.2.4
```

---

Actually, this file is the absolute wrong place for lower bounds. Its semantic purpose is to hold temporary exclusions, which this isn't.
The entry should be deleted instead.
```suggestion
```